libs/core/src/scss/_global-styles.scss

```diff
@@ -310,3 +311,11 @@ html.ios ion-modal.kirby-modal-v2 {
     display: flex;
   }
 }
+
+.swiper-pagination-bullet {
+  border-radius: 2px;
+}
+
+div.pagination span.swiper-pagination-bullet:last-of-type {
```

Comment:
I totally agree, but for some reason I cannot make it work if I place the styling in `carousel.component.scss`. It's not because the styling is overridden, but more like it is never applied to the bullet `span` elementens. Do you have any ideas what the problem could be @RasmusKjeldgaard?